homework2/task5.py

```diff
@@ -0,0 +1,55 @@
+"""
+Some of the functions have a bit cumbersome behavior when we deal with
+positional and keyword arguments.
+
+Write a function that accept any iterable of unique values and then
+it behaves as range function:
+
+
+import string
+
+
+assert = custom_range(string.ascii_lowercase,'g') == /
+['a', 'b', 'c', 'd', 'e', 'f']
+assert = custom_range(string.ascii_lowercase, 'g', 'p') == /
+['g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o']
+assert = custom_range(string.ascii_lowercase, 'p', 'g', -2) == /
+['p', 'n', 'l', 'j', 'h']
+
+"""
+from typing import Iterable, List
+
+
+def custom_range(iterable: Iterable[any], iter_range: any, *args) -> List[any]:
```

Comment:
`make_custom_range(iterable, start, end=None, step=None)`
Always use verb in name and try to avoid generic naming for vars